aequilibrae/project/network/safe_class.py

```diff
@@ -33,3 +34,6 @@ def _save_new_with_geometry(self):
         data.extend([self.geometry.wkb, self.__srid__])
         sql = f'Insert into {self._table} ({",".join(up_keys)}) values({markers})'
         return data, sql
+
+    def conn(self):
```

Comment:
I'm a little unsure about calling this method conn - my intuition is that it would return an existing connection, not establish a new one.

Reply:
renamed to `connect_db`